src/ec/packages/ec-component-tag/tag.html.twig

```diff
@@ -0,0 +1,65 @@
+{#
+  Parameters:
+   - "tag" (associative array) (default: predefined structure): format:
+      {
+        type: 'link' (can be 'link', 'button', 'removable'),
+        path: '' (tag url if needed),
+        label: '' (tag text as string)
+      }
+    - "default_icon_path" (string) Default path for icon tag (need to render Icon component)
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+{% set _tag={
+    type: 'link',
+    path: '',
+    label: ''
+} %}
+
+{% if tag is defined %}
+  {% set _tag=_tag|merge(tag) %}
+{% endif %}
+
+{% set _css_class = 'ecl-tag' %}
+{% set _extra_attributes = '' %}
+
+{# Add extra class for removable tag #}
+{% if _tag.type == 'removable' %}
+  {% set _css_class=_css_class~' ecl-tag--removable' %}
+{% endif %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+{% if _tag.type!='link' %}
+  <button class="{{ _css_class }}" type="button"{{ _extra_attributes|raw }}>
+    {{ _tag.label }}
+    {% if _tag.type=='removable' %}
+      <span class="ecl-tag__icon">
+        {% include '../ec-component-icon/icon.html.twig' with { icon: { type: 'ui', name: 'close', size: 'xs', path: default_icon_path }, extra_classes: 'ecl-tag__icon-close' } %}
```

Comment:
We recently added the extra attribute `focusable="false"` to all svg icons, to prevent a bug on Internet Explorer. Could you add it on your side too?
See https://v2--europa-component-library.netlify.com/ec/components/icon/code/

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/17/commits/069ce0527105026e409f5ea3e42803bc88e2268b